src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,64 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String DELIMITER = "\\W+";
+    private static final String DELIMITER_FOR_BUILDER = ",";
+    private static final String BUY = "buy";
+    private static final String SUPPLY = "supply";
+    private static final String RESULT = "result";
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String read = readFromFile(fromFileName);
+        String convert = convertToString(read);
```

Comment:
don't use verbs for variable names, give more specific names.
Also, do not use such naming(builder, split, actualResult), how to guess what it stores?
```suggestion
        String data = readFromFile(fromFileName);
        String report = prepareReport(data);
```